sys/tools/syscalls/scripts/syscalls_map.lua

```diff
@@ -38,7 +38,7 @@ function syscalls_map.generate(tbl, config, fh)
 
 	for _, v in pairs(s) do
 		--print("num " .. v.num .. " name " .. v.name)
-		if v:native() and not v.type.NODEF and v.name ~= "yield" then
+		if v:native() and not v.type.NODEF and not v.type.NOLIB then
 			if v.name ~= "exit" and v.name ~= "vfork" then
```

Comment:
I've got a plan for these as well (dropping most `_foo` format symbols by adding an INTERPOSED flag), but that will generate more churn so I want to do it separately.